bricks/dart_frog_new/brick.yaml

```
@@ -0,0 +1,20 @@
+name: dart_frog_new
+description: A dart frog brick to create routes and middleware
+version: 0.1.0+1
+
+environment:
+  mason: ">=0.1.0-dev <0.1.0"
+
+vars:
+  route_path:
+    type: string
+    description: The path for the desired route
+    prompt: What is the path for the desired route?
+  type:
+    type: enum
+    description: the type of hing to be created
```

```suggestion
    description: the type of thing to be created
```

Is there another word we could use instead of "thing" that encapsulates these?

---

Stuff? :D

---

Gonna figure something out